cmd/lotus-bench/simple.go

```diff
@@ -266,7 +266,10 @@ var simplePreCommit1 = &cli.Command{
 			ProofType: spt(sectorSize, cctx.Bool("synthetic")),
 		}
 
-		var ticket [32]byte // all zero
+		ticket := [32]byte{}
```

Comment:
It might make sense that the ticket could be any arbitrary value, except all zeros.

Reply:
Like a user-defined value? Or just any random arbitrary value?

Reply:
From a proofs perspective it could be anything, hence a random value or something like all 1s would be fine. IIRC the value for the `ticket` comes from Drand when the code is normally run.